sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestRegisteredMappersWork.java

```diff
@@ -190,6 +191,13 @@ public void testNoErrorOnNoData() throws Exception
         List<Something> rs = bob.listAll();
         assertThat(rs).isEmpty();
 
+    }
+
+    @Test(expected = ResultSetException.class)
+    public void testIteratorCloses() throws Exception
+    {
+        Kabob bob = db.getJdbi().onDemand(Kabob.class);
+
         Iterator<Something> itty = bob.iterateAll();
         assertThat(itty.hasNext()).isFalse();
```

Comment:
Considering the expected exception for this test, it seems like the `assertThat` is superfluous, and all we really want is `itty.hasNext();`